### ISSUE TITLE
Problem installing in Linux

### ISSUE THREAD

**Issue by nuno-agostinho** [rvalieris/parallel-fastq-dump#3]
When trying to install the program using either `pip` or the `setup.py`, the following error appears:

```
Traceback (most recent call last):
  File "setup.py", line 8, in <module>
    open(pname).read().split("\n")
TypeError: list object is not an iterator
```

**Comment by rvalieris** [comment]
hello,

looks like an issue with python2, its a easy fix I will release an update soon, but I recommend using python3 if possible, as I haven't really tested this program with python2.

**Comment by rvalieris** [comment]
should be fixed now on the new release 0.5.5
